nova/scheduler/manager.py

```diff
@@ -135,6 +135,9 @@ def select_destinations(self, ctxt, request_spec=None,
                 raise exception.NoValidHost(reason=e.message)
 
             resources = utils.resources_from_request_spec(spec_obj)
+            is_resize = utils.request_is_resize(spec_obj)
+            if is_resize:
+                resources._ignore_consumers = instance_uuids
```

Comment:
This seems a bit intrusive. Maybe we should put it under a configurable flag that would enable / disable this functionality ? That can default to `true`.

Reply:
Why do you think it should be possible to disable this? Imho it's unexpected by customers, that a resize takes double the amount of resources and that there's  no space to shrink a VM because of that. So as long as the code works, I wouldn't say it makes sense to disable it?